_toc.yml

```
@@ -195,7 +195,7 @@
       title: Changing culture with the Dataset Maturity Model
       sections:
         - file: content/recipes/reusability/miappe #FCB061
-          title: Improving dataset maturity - the MIAPPE use case
+          title: MIAPPE-compliant plant genomic and genetic data submission to EMBL-EBI databases
```

```suggestion
          title: Improving dataset maturity - MIAPPE-compliant submission to EMBL-EBI databases
```

Shortening to accommodate UI constraints of the Table of Content side bar